include/geolib/sensors/DepthCamera.h

```diff
@@ -249,6 +249,8 @@ class DepthCamera {
 
     inline double getOpticalTranslationY() const { return ty_; }
 
+    inline bool initialized() const { return cache_valid_; }
```

Comment:
cache_ valid is niet hetzelfde als initialized.

Reply:
But it is a good representation. And useless to introduce a second boolean with the same behaviour.

Reply:
Could you add a docstring to define exactly what initialized means in this context.